core/src/main/java/bisq/core/payment/PaymentAccount.java

```diff
@@ -148,6 +148,20 @@ public TradeCurrency getSingleTradeCurrency() {
             return null;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        PaymentAccount that = (PaymentAccount) o;
+        return creationDate == that.creationDate &&
+                id.equals(that.id);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, creationDate);
+    }
+
```

Comment:
I would rather see a simple
```
@EqualsAndHashSode.Exclude
```
added to the `accountName` field rather than getting rid of lombok at all.

Reply:
`accountName` field is for test only I have added. In reality, we couldn't delete the object immediately after creation.(Not sure which field causes this)
Hence i feel it should be applied to entire class. pls let me know if otherwise.